src/drafts/MarkdownEditor/_ViewSwitch.tsx

```diff
@@ -18,39 +16,47 @@ type ViewSwitchProps = {
 // no point in memoizing this component because onLoadPreview depends on value, so it would still re-render on every change
 export const ViewSwitch = ({selectedView, onViewSelect, onLoadPreview, disabled}: ViewSwitchProps) => {
   // don't get disabled from context - the switch is not disabled when the editor is disabled
-  const {condensed} = useContext(MarkdownEditorContext)
 
-  const {label, icon, ...sharedProps} =
+  const sharedProps =
     selectedView === 'preview'
       ? {
-          variant: 'invisible' as const,
-          sx: {color: 'fg.default', px: 2},
           onClick: () => onViewSelect?.('edit'),
-          icon: PencilIcon,
-          label: 'Edit',
         }
       : {
-          variant: 'invisible' as const,
-          sx: {color: 'fg.default', px: 2},
           onClick: () => {
             onLoadPreview()
             onViewSelect?.('preview')
           },
           onMouseOver: () => onLoadPreview(),
           onFocus: () => onLoadPreview(),
-          icon: EyeIcon,
-          label: 'Preview',
         }
 
   return (
     <Box sx={{display: 'flex', flexDirection: 'row'}}>
-      {condensed ? (
-        <IconButton {...sharedProps} disabled={disabled} icon={icon} aria-label={label} />
-      ) : (
-        <Button {...sharedProps} leadingVisual={icon} disabled={disabled}>
-          {label}
-        </Button>
-      )}
+      <TabNav aria-label="Main">
```

Comment:
@ichelsea - is this a safe usage of `aria-label`? Or do we need the text in a visually hidden element that we point to using `aria-labelledby`?

Reply:
I'm not well versed in .tsx, is this ultimately a button? Where is it located/what does it do?

Reply:
I took this from https://primer.style/react/TabNav which mentions `Attention: Make sure to properly label your TabNav with an aria-label to provide context about the type of navigation contained in TabNav.`

Thinking about it, would `View mode` be a more descriptive label than `Main` ? I assume a screen reader user would benefit from understanding `View mode: Write` or `View mode: Preview`?

Reply:
@ichelsea this is used in the MarkdownEditor for the TabNav for `Write` and `Preview`.

Reply:
Ah, gotcha! So our `<TabNav>` is essentially a `role="tablist"` under the hood? We do want it to have an accessible name, so either `aria-label` or `aria-labelledby` works here. 

"View mode" is definitely a step in the right direction vs. "main", especially because "main" is used in main landmark identification. Something about it still feels like it needs adjustment, but I don't have a better suggestion at the moment.

Reply:
I updated the text string to use `View mode` now and will merge in given the existing approvals. Given the discussion around the Rails CommentBox I do anticipate some changes down the road for the Rails component as well as this one as we want to make sure it stays aligned. The React component is already being used in the Memex Sidesheet (and the new Issues and PR experiences) if you do want to check it out once it's merged 👍